integration-tests/cucumber/cucumber.spec.js

```diff
@@ -232,353 +230,402 @@ versions.forEach(version => {
+                  .content
+                assert.exists(testSession.metrics[TEST_CODE_COVERAGE_LINES_PCT])
+
+                const eventTypes = eventsRequest.payload.events.map(event => event.type)
+                assert.includeMembers(eventTypes, ['test', 'test_suite_end', 'test_module_end', 'test_session_end'])
+                const numSuites = eventTypes.reduce(
+                  (acc, type) => type === 'test_suite_end' ? acc + 1 : acc, 0
+                )
+                assert.equal(numSuites, 2)
+              }).catch(done)
 
-            receiver.assertPayloadReceived(({ payload }) => {
-              const eventTypes = payload.events.map(event => event.type)
-              // because they are not skipped
-              assert.includeMembers(eventTypes, ['test', 'test_suite_end', 'test_module_end', 'test_session_end'])
-              const numSuites = eventTypes.reduce(
-                (acc, type) => type === 'test_suite_end' ? acc + 1 : acc, 0
+              childProcess = exec(
+                runTestsWithCoverageCommand,
+                {
+                  cwd,
+                  env: envVars,
+                  stdio: 'pipe'
+                }
               )
-              assert.equal(numSuites, 2)
-              const testSession = payload.events.find(event => event.type === 'test_session_end').content
-              assert.propertyVal(testSession.meta, TEST_ITR_TESTS_SKIPPED, 'false')
-              assert.propertyVal(testSession.meta, TEST_CODE_COVERAGE_ENABLED, 'true')
-              assert.propertyVal(testSession.meta, TEST_ITR_SKIPPING_ENABLED, 'true')
-              const testModule = payload.events.find(event => event.type === 'test_module_end').content
-              assert.propertyVal(testModule.meta, TEST_ITR_TESTS_SKIPPED, 'false')
-              assert.propertyVal(testModule.meta, TEST_CODE_COVERAGE_ENABLED, 'true')
-              assert.propertyVal(testModule.meta, TEST_ITR_SKIPPING_ENABLED, 'true')
-            }, ({ url }) => url.endsWith('/api/v2/citestcycle')).then(() => done()).catch(done)
-
-            childProcess = exec(
-              runTestsWithCoverageCommand,
-              {
-                cwd,
-                env: envVars,
-                stdio: 'inherit'
-              }
-            )
-          })
-          it('does not skip tests if test skipping is disabled by the API', (done) => {
-            receiver.setSettings({
-              code_coverage: true,
-              tests_skipping: false
+              childProcess.stdout.on('data', (chunk) => {
+                testOutput += chunk.toString()
+              })
+              childProcess.stderr.on('data', (chunk) => {
+                testOutput += chunk.toString()
+              })
+              childProcess.on('exit', () => {
+                // check that reported coverage is still the same
+                assert.include(testOutput, runTestsWithCoverageCommandOutput)
```

Comment:
I think it's fine to just use `Lines        : 100%` here and remove `runTestsWithCoverageCommandOutput`. The two file difference tells us little